tabular/src/autogluon/tabular/learner/default_learner.py

```diff
@@ -179,9 +179,9 @@ def general_data_processing(self, X: DataFrame, X_val: DataFrame, X_unlabeled: D
             if self.feature_generator.is_fit():
                 logger.log(20, f'{self.feature_generator.__class__.__name__} is already fit, so the training data will be processed via .transform() instead of .fit_transform().')
                 X_super = self.feature_generator.transform(X_super)
-                self.feature_generator.print_feature_metadata_info()
             else:
                 X_super = self.feature_generator.fit_transform(X_super)
+            self.feature_generator.print_feature_metadata_info()
```

Comment:
Change should be reverted, metadata is printed internally in fit_transform which is why we don't want to print it twice.